processing/src/test/java/io/druid/query/groupby/epinephelinae/BufferHashGrouperTest.java

```diff
@@ -48,6 +53,16 @@
   @Rule
   public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+  private MappedByteBuffer buffer;
+
+  @After
+  public void tearDown()
+  {
+    if (buffer != null) {
+      ByteBufferUtils.unmap(buffer);
```

Comment:
Can this use a `io.druid.segment.CloserRule` `@Rule` to do this?

Reply:
It could, although it seems overkill.

Reply:
I changed it to use CloserRule anyway.